app/models/listing.rb

```diff
@@ -55,7 +55,7 @@ class Listing < ApplicationRecord
   # @return [TrueClass] if the Listing is enabled for this Forem
   # @return [FalseClass] if the Listing is disabled for this Forem
   def self.feature_enabled?
-    FeatureFlag.accessible?(:listing_feature_enabled)
+    FeatureFlag.accessible?(:listing_feature)
```

Comment:
Ah, now I see why you were talking about explicitly disabling. I've only ever used `enabled?` (either standalone for globally or percentage-enabled features) or with an associated actor. And it defaults to `false` unless explicitly enabled, which removes the need to create or explicitly disable the flag anywhere:

```ruby
FeatureFlag.enabled? :brand_new_feature
# => false
FeatureFlag.enable :brand_new_feature
# => true
FeatureFlag.enabled? :brand_new_feature
# => true
```

I'm sure `accessible?` has its uses (otherwise it probably wouldn't exist), but I don't know what they are. This feature flag (and most I've dealt with) seem perfect for `enabled?`, though — if we only want to enable this feature on DEV, this PR becomes a 1-line change by switching to `enabled?` [adding this feature flag to DEV manually](https://dev.to/admin/feature_flags/features) pre-deployment so there won't be a gap in functionality.

Reply:
Based on conversations offline and thinking about this again the following steps make the most sense:

Amend this pull request to:
  * Flip `accessible?` to `enabled?`
  * Remove the migration
  * Adjust the commit message and body

Before we would merge, we would want a stable feature flag name (I misnamed it originally).  We'd want to communicate the timeline for disabling the listings.  We'd also want to turn on the flag on DEV.  Then with those all confirmed, merge the pull request.

Reply:
Reading notes and a reminder for Monday: One challenge of moving from `accessible?` to `enabled?` is that we'd need to update the test ecosystem to default to "on" (e.g. we have lots of listing tests)

Reply:
@jeremyf One thing I was also thinking about was adding a new type of RSpec metadata that will allow us to write specs with features enabled or disabled, something like this:

```ruby
it "does something amazing when the feature flag is one", feature_flags: { flag1: true, flag2: false } do
```

which would translate to something like 

```ruby
example.metadata[:feature_flags].each do |flag, status|
  expect(FeatureFlag).to receive(:enabled).with(flag).and_return(status)
end
```

I think I'd like to tackle that after my current WIP feature flag work.